docs/USAGE-GUIDE.md

```diff
@@ -1560,6 +1568,41 @@ handshake.
 
 **s2n_connection_is_session_resumed** returns 1 if the handshake was abbreviated, otherwise returns 0, for tls versions < TLS1.3.
 
+## Additional TLS1.3 Session Resumption Related Calls
+
+Session resumption works differently in versions TLS1.3 and higher. Session ticket messages are now sent after the handshake in "post-handshake" messages, and multiple session tickets may be issued for the same connection. While some of the TLS1.2 session resumption APIs are still relevant for TLS1.3 session resumption, additional APIs are needed to utilize all the capabilities of TLS1.3 session resumption.
+
+```c
+int s2n_config_set_initial_ticket_count(struct s2n_config *config, uint8_t num);
+int s2n_connection_add_new_tickets_to_send(struct s2n_connection *conn, uint8_t num);
+int s2n_connection_set_server_keying_material_lifetime(struct s2n_connection *conn, uint32_t lifetime_in_secs);
+
+typedef int (*s2n_session_ticket_fn)(struct s2n_connection *conn, void *ctx, struct s2n_session_ticket *ticket);
+int s2n_config_set_session_ticket_cb(struct s2n_config *config, s2n_session_ticket_fn callback, void *ctx);
+int s2n_session_ticket_get_data_len(struct s2n_session_ticket *ticket, size_t *data_len);
+int s2n_session_ticket_get_data(struct s2n_session_ticket *ticket, size_t max_data_len, uint8_t *data);
+int s2n_session_ticket_get_lifetime(struct s2n_session_ticket *ticket, uint32_t *session_lifetime);
+```
```

Comment:
Could we follow the same format that we do for the other API descriptions in the Usage Guide. It seems like Session Resumption APIs are the outlier. For API descriptions the rest of the Usage Guide follows: 

- API Title 
- API Snippet 
- Description 

For ex: https://github.com/aws/s2n-tls/blob/main/docs/USAGE-GUIDE.md#s2n_config_new

```
### s2n\_config\_new

```c
struct s2n_config * s2n_config_new();
 `` `
 
s2n_config_new** returns a new configuration object suitable for associating certs and keys.
This object can (and should) be associated with many connection objects. 
```

Reply:
Idk, we're not consistent, especially for chunks of closely related APIs. For example, https://github.com/aws/s2n-tls/blob/main/docs/USAGE-GUIDE.md#s2n_mem_set_callbacks, https://github.com/aws/s2n-tls/blob/main/docs/USAGE-GUIDE.md#s2n_connection_set_send_cb, https://github.com/aws/s2n-tls/blob/main/docs/USAGE-GUIDE.md#asynchronous-private-key-operations-related-calls.

Let's do what works / is readable.

Reply:
Yeah, I'm trying to follow the format of the TLS1.2 session resumption docs so they fit together cohesively. I would prefer if we just stuck to one style though.